src/main/java/io/confluent/kafkarest/ProducerPool.java

```diff
@@ -143,7 +144,8 @@ private static String getBootstrapBrokers(ZkUtils zkUtils) {
         if (bootstrapBrokers.length() > 0) {
           bootstrapBrokers += ",";
         }
-        bootstrapBrokers += ep.connectionString();
+        String hostport = ep.host() == null ? ":" + ep.port() : Utils.formatAddress(ep.host(), ep.port());
+        bootstrapBrokers += ep.securityProtocol() + "://" + hostport;
```

Comment:
NM.  Found it:  https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/utils/Utils.java#L52-L54

Question:  any particular reason to add the protocol prefix?